loader/standard_loader.py

```diff
@@ -198,11 +198,14 @@ def load_all_bundles(self, input_json: typing.List[dict]):
                 - len(self.bundles_loaded)
             if bundles_unattempted:
                 logger.warning(f'Did not yet attempt to load {bundles_unattempted} bundles')
+                success = False
             if len(self.bundles_failed_unparsed) > 0:
                 logger.error(f'Could not parse {len(self.bundles_failed_unparsed)} bundles')
+                success = False
             if len(self.bundles_failed_parsed) > 0:
                 logger.error(f'Could not load {len(self.bundles_failed_parsed)} bundles')
+                success = False
                 # TODO: ADD COMMAND LINE OPTION TO SAVE ERROR LOG TO FILE https://stackoverflow.com/a/11233293/7830612
                 logger.info(f'Successfully loaded {len(self.bundles_loaded)} bundles')
```

Comment:
This line should be removed from here and moved into the `if success` block, replacing the line that is currently there.

Reply:
I agree it should be moved, but not there. The intention of this line was to tell the user exactly how many bundles were *actually* loaded, but only when it's different from the total number of bundles. It belongs in an else statement for success.